paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/MainActivity.kt

```diff
@@ -147,6 +158,11 @@ private fun MainScreen(items: List<MenuItem>) {
             items = groupedItems.getOrElse(MenuItem.Section.Internal) { emptyList() },
         )
 
+        Section(
+            title = "\uD83C\uDFD7 Wallet Mode (under construction)",
```

Comment:
It already says `under construction` below, so I don’t think we need this.

```suggestion
            title = "Wallet Mode",
```